core/modules/mailer/store/index.ts

```diff
@@ -25,12 +26,18 @@ export const mailerStore: Module<any, any> = {
                 })
               })
                 .then(res => resolve(res))
-                .catch(() => reject())
```

Comment:
can we refactor it to async/await form? @patzick will buy you a beer for that ;p

Reply:
Sure, everything for beer! :D 🍻